pages/classes/index.js

```diff
@@ -76,6 +79,19 @@ export default function Classes({
 }) {
   return (
     <>
+      <ToastContainer
```

Comment:
This <ToastContainer> is not imported from the components/toastContainer.js

The idea is you create a component under components that wraps the original ToastContainer to hide a lot of these options that we don't really want to configure every single time we want to add a ToastContainer. 
Let's call it "ClassroomToastContainer".

I don't think we need to configure options "position", closeonclick, hideProgressBar", "draggable" "rtl"... etc

I think we should be able only to pass in a few args, like color or whether the alert is a success and failure.

I don't know the exact answer to this problem, but essentially could you all go back and think about how to make this `<ToastContainer>` and `toast.success(` call to be as simple as possible for developers in the future? @GuillermoFloresV and I eventually want to refactor all of the alert(s to use your framework you are laying out here.